modules/test/playwright/tests/journal-web/journalArticleAutosave.spec.ts

```diff
@@ -60,7 +60,6 @@ const autoSaveUndoRedoTest = mergeTests(
 	featureFlagsTest({
 		'LPD-11228': true,
 		'LPD-15596': true,
-		'LPD-36053': true,
```

Comment:
Now this is the same than `autoSaveTest`, so I would remove `autoSaveUndoRedoTest` and replace all its uses by  `autoSaveTest`

Reply:
I pushed up another commit to remove autoSaveUndoRedoTest